marklogic-data-hub-central/ui/src/components/queries/queries.tsx

```diff
@@ -202,12 +202,14 @@ const Query = (props) => {
             toggleSaveChangesIcon(false);
             toggleDiscardIcon(false);
         }
+
+        // setColumnSelectorTouched(JSON.stringify(searchOptions.selectedTableProperties) !== JSON.stringify(props.columns))
```

Comment:
Can we delete these commented out lines.

Reply:
Removed.